docs/tutorials/development-setup.md

```diff
@@ -181,64 +194,51 @@ pytest tests/integration/test_whisper.py
 
 ### Manual Testing
 
-```bash
-# Test voice conversation
-voice-mode converse --debug
+#### Testing with MCP
 
-# Test specific tool
-voice-mode test-tool converse
+To test your development version with MCP (Model Context Protocol):
 
-# Test with different providers
-VOICEMODE_TTS_BASE_URLS=http://localhost:8880/v1 voice-mode converse
-```
+1. **Update `.mcp.json` to point to your development version**
+   ```json
+   {
+     "mcpServers": {
+       "voice-mode": {
+         "command": "uv",
+         "args": ["run", "voice-mode", "mcp"]
+       }
+     }
+   }
+   ```
 
-## Contributing
+2. **Run `mcp` to test the connection**
+   ```bash
+   mcp test voice-mode
```

Comment:
What is `mcp` command?

Reply:
removed in https://github.com/mbailey/voicemode/commit/4573d25a8bb736ed75f970d409383ce34272f95c.